File: Python/bubble_sort.py
def sort(list):
    for j in range(1, len(list), 1):
        for i in range(len(list)-j):
            if list[i] > list[i+1]:
                temp = list[i]
                list[i] = list[i+1]
                list[i+1] = temp
    return list

File: C++/bubble_sort.cpp
void bubble_sort(int *tab, int n){
    int temp;
    for(int i = 0; i < n; i++){
        for(int j = 0; j < n - i - 1; j++) {
            if (tab[j + 1] < tab[j]) {
                temp = tab[j];
                tab[j] = tab[j+1];
                tab[j+1] = temp;
            }
        }
    }
}
